src/main/features/claimant-response/views/check-and-send.njk

```diff
@@ -111,6 +111,20 @@
 
     {{ tableEnd() }}
 
+    {{ tableStart('judgmentRequest', 'Judgment request') }}
+
+      {{ row('Has the defendant paid some of the amount owed?',
+      draft.paidAmount.option.value | capitalize, changeLink = paidAmountUrl, bottomBorder = false, bold = true) }}
```

Comment:
Please use path directly without passing it from route e.g. `changeLink = ClaimantResponsePaths.somePage.evaluateUri({.externalId: claim.externalId })`

Reply:
Done